build/DependencyVersions.props

```diff
@@ -17,7 +17,7 @@
 
     <CLI_NuGet_Version>4.4.0-preview1-4434</CLI_NuGet_Version>
     <CLI_NETStandardLibraryNETFrameworkVersion>2.0.0-preview3-25514-04</CLI_NETStandardLibraryNETFrameworkVersion>
-    <CLI_WEBSDK_Version>2.0.0-rel-20170629-588</CLI_WEBSDK_Version>
```

Comment:
SDK version looks odd, but most of the time the version looks like https://github.com/dotnet/cli/commit/53f6fe597f305913bfa691fb5355b3b06326e300

Reply:
See below. That's because this is a different component. It's what puts the contents of https://github.com/aspnet/websdk in to the CLI not https://github.com/dotnet/sdk